### PR TITLE
Server improvements + Extra Attribute display

### PR DESCRIPTION
**Pull request by fosskers**
In learning the server code, I've updated it to:
- Yield 404 on a tile outside the current bounds, instead of an ErrorTile (Rob's suggestion)
- Use less verbose naming conventions
- Hold route definitions and their handle code in separate functions for easier readability

Note that even with an ErrorTile/404 returned on a Tile outside the current bounds, Leaflet calling these in the first place feels wasteful. Something to consider for the next sprint.

### CODE REVIEW

File: server/src/main/scala/geotrellis/admin/server/AdminActor.scala [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+    }
+  }
+
+  val missingTileHandler = ExceptionHandler {
+    case ex: TileNotFoundError => respondWithStatus(404) {
+      complete(s"No tile: ${ex.getMessage}")
+    }
+  }
+
+  def tms = handleExceptions(missingTileHandler) {
```
Comment (moradology) [inline server/src/main/scala/geotrellis/admin/server/AdminActor.scala:168]: This is definitely a good change. Much cleaner

File: client/src/main/resources/index-dev.html [hunk 2]
```diff
@@ -31,15 +31,15 @@
...
-  <script>
-    geotrellis.admin.client.Main().main();
-  </script>
+
+  <!-- Include Scala.js compiled code -->
+  <script type="text/javascript" src="../../../target/scala-2.11/geotrellis-admin-client-fastopt.js"></script>
+  <script type="text/javascript" src="../../../target/scala-2.11/geotrellis-admin-client-jsdeps.js"></script>
+
+  <!-- Run the App -->
+  <script type="text/javascript" src="../../../target/scala-2.11/geotrellis-admin-client-launcher.js"></script>
```
Comment (moradology) [inline client/src/main/resources/index-dev.html:42]: Much nicer.

**Comment by moradology** [comment]
This looks good - merging in

File: client/src/main/resources/index-dev.html (hunk 2 above)
Comment (fosskers) [inline client/src/main/resources/index-dev.html:42]: But still broken somehow, as I mentioned in Gitter.

**Pull request merged**